tests/auto/rt_auto.py

```diff
@@ -0,0 +1,294 @@
+"""Automation of UFS Regression Testing
+
+This script automates the process of UFS regression testing for code managers
+at NOAA-EMC
+
+This script should be started through rt_auto.sh so that env vars are set up
+prior to start.
+"""
+from __future__ import print_function
+from github import Github as gh
+import argparse
+import datetime
+import time
+import threading
+import subprocess
+import re
+import os
+import logging
+
+class GHInterface:
+    '''
+    This class stores information for communicating with GitHub
+    ...
+
+    Attributes
+    ----------
+    GHACCESSTOKEN : str
+      API token to autheticate with GitHub
+    client : pyGitHub communication object
+      The connection to GitHub to make API requests
+    '''
+    def __init__(self) -> None:
+        self.logger = logging.getLogger('GHINTERFACE')
+        try:
+            with open('accesstoken.txt', 'rb') as f:
```

Comment:
I think we should use the env. variable `GITHUB_TOKEN` rather than read that from a file.
That is the way github recommends it.

Reply:
so where should we export/store that env var? isn't it the same as having accesstoken.txt? Both still end up in a text file somewhere...

Reply:
That is correct, though accessing it as an environment variable of `GITHUB_TOKEN` adheres to the GitHub intended use.
Leave as is for now.

Reply:
Is it possible to store the token as a secret of the ufs-weather-model repository?